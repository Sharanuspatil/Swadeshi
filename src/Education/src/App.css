* {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
    font-family: 'PT Sans', sans-serif;
  }
  
  .home,
  .auction,
  .education,
  .blog,
  .uploadvideo,
  .signup {
    display: flex;
    height: 90vh;
    align-items: center;
    justify-content: center;
    font-size: 3rem;
  }
  
  .auction {
    background-image: url('images/image1.jpeg');
    background-position: center;
    background-size: cover;
    background-repeat: no-repeat;
    color: #fff;
    font-size: 100px;
  }

  .education {
    background-image: url('images/image2.jpeg');
    background-position: center;
    background-size: cover;
    background-repeat: no-repeat;
    color: #fff;
    font-size: 100px;
  }
  
  
  .blog {
    background-image: url('images/image3.jpeg');
    background-position: center;
    background-size: fill;
    background-repeat: no-repeat;
    color: #fff;
    font-size: 100px;
  }
  

  .signup {
    background-image: url('images/image4.jpeg');
    background-position: center;
    background-size: cover;
    background-repeat: no-repeat;
    color: #fff;
    font-size: 100px;
  }
  
 