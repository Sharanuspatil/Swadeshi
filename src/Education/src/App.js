
import React from 'react';
import {BrowserRouter as Router,Switch,Route} from 'react-router-dom';
import './App.css';
import Home from './components/pages/Home';
function App() {
  return (
    <Router>
     
      <Switch>
        <Route path='/Education' exact component= {Home} />
      </Switch>
    </Router>
      
    
  );
}

export default App;
