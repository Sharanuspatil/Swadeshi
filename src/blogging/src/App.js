import "./App.css";
import Home from "./pages/home/Home";
import Single from "./pages/single/Single";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
function App() {
  const user = true;
  return (
    <Router>
      <Switch>
        <Route exact path="/Blog">
          <Home />
        </Route>

        <Route path="/post/:postId">
          <Single />
        </Route>
      </Switch>
    </Router>
  );
}

export default App;
