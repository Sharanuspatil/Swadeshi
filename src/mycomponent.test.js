import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
configure({ adapter: new Adapter() });

import App from './App';
describe('App', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<App debug />);
  
    expect(component).toMatchSnapshot();
  });
});
