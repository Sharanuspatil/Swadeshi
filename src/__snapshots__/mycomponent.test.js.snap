// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
